Packages/Core/Editor/Drawers/VariableDrawer.cs

```diff
@@ -14,15 +14,29 @@ public override void OnGUI(Rect position, SerializedProperty property, GUIConten
                 return;
             }
 
+            label = EditorGUI.BeginProperty(position, label, property);
```

Comment:
Sorry, but need you to revert these changes before I merge this PR (the complete file).